book/modules/appendix1.tex

```diff
@@ -71,15 +71,29 @@
 		and the system of linear equations corresponding to the vector equation $t\vec w+\vec u=r\vec v$ where $\vec w=\mat{1\\-1}$, $\vec u=\mat{2\\3}$,
 		and $\vec v=\mat{4\\4}$.
 
-		XXX Finish
+		Paying attention to the coordinates and treating $x$, $y$, and $z$ as coefficients\index{Linear combination!coefficient of}, we can see that $\systeme{2x+3y+z=2,0x+y-z=-1}$ corresponds to the vector equation
```

Comment:
-> `The system $\systeme{2x+3y+z=2,0x+y-z=-1}$ corresponds to the vector equation`

I.e., cut the explanation.